mpiPi.h.in

```diff
@@ -29,7 +29,25 @@
 #include "mpiPconfig.h"
 
 #ifdef ENABLE_BFD
-#include "bfd.h"
+  #ifndef PACKAGE
```

Comment:
Can this be shortened to
```C

#ifndef PACKAGE
#define PACKAGE
#define MPIP_PACKAGE_UNDEF
#endif
#ifndef PACKAGE_VERSION
#define PACKAGE_VERSION
#define MPIP_PACKAGE_VERSION_UNDEF
#endif
#include "bfd.h"

#ifdef MPIP_PACKAGE_UNDEF
#undef PACKAGE
#undef MPIP_PACKAGE_UNDEF
#endif 

#ifdef MPIP_PACKAGE_VERSION_UNDEF
#undef PACKAGE_VERSION
#undef MPIP_PACKAGE_VERSION_UNDEF
#endif 
```

Though not much shorter it's more straightforward and easier to verify.